ACKNOWLEDGMENTS.md

```diff
@@ -12,4 +12,3 @@ MLX Examples was developed with contributions from the following individuals:
 - Shunta Saito: Added support for PLaMo models.
 - Gabrijel Boduljak: Implemented `CLIP`.
 - Markus Enzweiler: Added the `cvae` examples.
-- Rasmus Kinnunen: Fixed a security hole in the `llms/mlx_lm` example
```

Comment:
Turns out I was only using about half of my braincells yesterday and got BaseHTTPRequestHandler and SimpleHTTPRequestHandler mixed up, BaseHTTPRequestHandler doesn’t actually serve the local directory by default… awkward 🫠 Will be working overtime to make up for it